ext/gd/gd.c

```diff
@@ -67,6 +68,7 @@ static int le_gd, le_gd_font;
 #include <gdfontmb.h> /* 3 Medium bold font */
 #include <gdfontl.h>  /* 4 Large font */
 #include <gdfontg.h>  /* 5 Giant font */
+#include <zend.h>
```

Comment:
Why is this included? `#include <php.h>` already includes this and is the general conversion